cypress/support/config/settings.js

```diff
@@ -6961,7 +6961,7 @@ module.exports = () => ({
           },
           local: {
             paths: ['/thai/international-23252825'],
-            enabled: true,
+            enabled: false,
```

Comment:
Is there a reason why this is now disabled locally?

Reply:
@andrewscfc suggested to disable it as it was passing locally and on the deployment pipeline. To keep us out of blocked.